README.md

```diff
@@ -19,6 +19,18 @@ Cargo.toml:
 onvif-rs = "0.1"
 ```
 
+## Build
+
+If you have an issue with OpenSSL build under Ubuntu, perform the following actions:
+
+```
+sudo apt install openssl1.0 libssl1.0.0 libssl1.0-dev
+cargo clean
+OPENSSL_LIB_DIR="/usr/lib/x86_64-linux-gnu"
+OPENSSL_INCLUDE_DIR="/usr/include/openssl"
```

Comment:
Do you need `export` here?

Reply:
You are right, in a command line it should be done with export. I'll fix text.